Doc/library/dataclasses.rst

```diff
@@ -185,7 +185,10 @@ Module contents
    - ``slots``: If true (the default is ``False``), :attr:`~object.__slots__` attribute
      will be generated and new class will be returned instead of the original one.
      If :attr:`~object.__slots__` is already defined in the class, then :exc:`TypeError`
-     is raised.
+     is raised. Calling no-arg ``super()`` in dataclasses using ``slots=True`` will result in
+     the following Exception being raised:
+     ``TypeError: super(type, obj): obj must be an instance or subtype of type``.
+     The two-arg ``super()`` is a valid workaround. See :issue:`46404` for full details.
```

Comment:
Would it be better to use `:gh:` instead of `:issue:` to link directly to the GitHub issue? I guess it doesn't really matter though since it redirects from bpo anyway.

Reply:
I just copied this from another issue link I found in the docs :man_shrugging: 

Happy to do whatever makes the most sense.

Reply:
Yep, we should use the GH form:

```suggestion
     The two-arg ``super()`` is a valid workaround. See :gh:`90562` for full details.
```